portfolio/src/main/java/com/google/sps/servlets/CommentForumServlet.java

```diff
@@ -59,6 +59,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Parse input from the form
     String commentString = getParameter(request, "text-input", "");
+    String userName = getParameter(request, "user-name", "Anonymous");
     boolean upperCase = Boolean.parseBoolean(getParameter(request, "upper-case", "false"));
     boolean lowerCase = Boolean.parseBoolean(getParameter(request, "lower-case", "false"));
     long timestamp_ms = System.currentTimeMillis();
```

Comment:
nit: Be consistent with variable naming style. Our java style guide mandates lowerCamelCase, not snake_case